catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -199,7 +199,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
-            <version>1.13</version>
+            <version>1.14</version>
```

Comment:
I'm not sure 🤷‍♀️ . I thought I'd upgrade it just in case.
https://github.com/apache/karaf/commit/4ea71de4da9b9991f2a53ef4f3f1942b04f35698